BeagleBone/TachController/pru/main.c

```diff
@@ -32,15 +32,17 @@ static const uint32_t k_host_interupt = 0x1 << 30;
 // PRU0 uses system event 16 (To ARM) and 17 (From ARM)
 // PRU1 uses system event 18 (To ARM) and 19 (From ARM)
 
-static const uint32_t k_pru_to_host_event = 16;
-static const uint32_t k_pru_from_host_event = 17;
+static const uint32_t k_pru_to_host_event = 18;
+static const uint32_t k_pru_from_host_event = 19;
 
```

Comment:
Arggg, I had this switched to 18 and 19 at one point, I totally remember switching when I decided to use PRU1.  Dang it.  I swear I checked all these magic numbers.  

Thanks!